### PR TITLE
Split scripts for reducing time

### PR DESCRIPTION
**Pull request by Progi1984**

### CODE REVIEW

File: .github/workflows/pr_test.yml [hunk 1]
```diff
@@ -197,9 +221,10 @@ jobs:
         run: npm install
         working-directory: '${{ env.PS_DIR }}/tests/UI'
 
-      - name: Run tests
+      - name: Run tests 8.0
+        if: ${{ env.BASE_BRANCH != 'develop' }}
```
Comment (0x346e3730) [inline .github/workflows/pr_test.yml:225]: Why don't we want to run 8.0 tests for develop branch ?

Comment (Progi1984) [reply .github/workflows/pr_test.yml:225]: Modifications to package.json for UI Tests are only applied on `develop` branch.

File: .github/workflows/pr_test.yml [hunk 2]
```diff
@@ -208,8 +233,23 @@ jobs:
           HEADLESS: true
           TAKE_SCREENSHOT_AFTER_FAIL: true
 
-      - run: echo "MY_COMMAND=$( echo -e '${{ matrix.TEST_COMMAND }}' | tr  ':' '-'  )" >> $GITHUB_ENV
-        if: ${{ always() }}
+      - name: Run tests develop
+        if: ${{ env.BASE_BRANCH == 'develop' }}
+        id: my_test
```
Comment (0x346e3730) [inline .github/workflows/pr_test.yml:238]: I think this is the produce of a copy/paste as the id seems pretty generic, could you give it a better name ?

**Review by 0x346e3730 — CHANGES_REQUESTED** [review]

File: .github/workflows/pr_test.yml (hunk 2 above)
Comment (Progi1984) [reply .github/workflows/pr_test.yml:238]:
```suggestion
```

Comment (Progi1984) [reply .github/workflows/pr_test.yml]: Fixed

**Review by 0x346e3730 — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Pull request merged**